packages/ui-kit/src/components/TimeSeries/TimeSeries.types.ts

```diff
@@ -29,7 +29,13 @@ export interface TimeSeriesChartProps {
   fillArea?: boolean
 }
 
-export interface TimeSeriesBaseProps extends ErrorFallbackProps, DataComponentProps {
+export interface TimeSeriesBaseProps extends Omit<ErrorFallbackProps, 'error'>, DataComponentProps {
+  /** @deprecated This type is deprecated, use `errorFallbackProps` and `errorFallback` instead */
+  error?: {
+    title: string
+    body: string
+  } | null
+
```

Comment:
Why not just extend to `ErrorFallbackProps` and deprecate on `ErrorFallbackProps`? Is there somewhere we don't want to deprecate it?

```suggestion
export interface TimeSeriesBaseProps extends ErrorFallbackProps, DataComponentProps {
```

Reply:
We want to keep `error` props as a part of `ErrorFallbackProps` but we want to deprecate it here (in favor of `errorFallback`). But I see how to improve this part.